internal/handlers/tigris/tigris.go

```diff
@@ -34,8 +34,11 @@ func notImplemented(command string) error {
 
 // NewOpts represents handler configuration.
```

Comment:
Can we at least add more docs? E.g. are all the params required and where one can find what is each param needed for (maybe we can link some Tigris docs)?